fabric-textures-v0/src/main/java/net/fabricmc/fabric/impl/client/texture/ColorSniffer.java

```diff
@@ -0,0 +1,52 @@
+package net.fabricmc.fabric.impl.client.texture;
```

Comment:
If this is API it should be in the api package, not impl

Reply:
I wasn't sure about putting it in .api since it's usable out of the box

Reply:
if mods can touch it then it's gotta go in .api